root/static/scripts/common/components/TagEditor.js

```diff
@@ -436,158 +436,166 @@ class TagEditor extends React.Component<TagEditorProps, TagEditorState> {
   }
 }
 
-class MainTagEditor extends TagEditor {
```

Comment:
The diff here is absolutely nuts for some reason, but: this moves the hydrate from the export at the end to here, `export const MainTagEditor = hydrate<TagEditorProps>(whatever)` etc. Every change inside the class extends is indentation.